### PR TITLE
Dev K8S deployment YAML etc

### PR DESCRIPTION
**Pull request by NickGrantX**
# Motivation and Context
We need to be able to deploy the Load Generator into GCP/K8S.

# What has changed
Added various bits to be able to deploy & run the Load Generator in a dev cluster in GCP.

# How to test?
Follow the README doc for how to deploy to GCP.

# Links
Trello: https://trello.com/c/o8tdHgx9

### CODE REVIEW

**Comment by NickGrantX** [comment]
[Create Load Generator YAML Deployment](https://trello.com/c/o8tdHgx9/909-create-load-generator-yaml-deployment)

File: Makefile [hunk 1]
```diff
@@ -8,4 +8,13 @@ flake:
...
+	docker build -t eu.gcr.io/census-rm-ci/rm/census-rm-load-generator .
+
+delete-pod:
+	kubectl delete deploy load-generator
+
+apply-deployment:
+	kubectl apply -f load-generator-deployment.yml
+
+connect-to-pod:
+	kubectl exec -it `kubectl get pods -o name | grep -m1 load-generator | cut -d'/' -f 2` -- /bin/bash
```
Comment (AdamHawtin) [inline Makefile]:
```suggestion
	 kubctl exec -it $(kubectl get pods --selector=app=load-generator -o jsonpath='{.items[*].metadata.name}') -- /bin/bash
```

**Review by AdamHawtin — CHANGES_REQUESTED** [review]
Looks fine, just one shell command can be done with more `kubctl`, less `sh`.

**Review by AdamHawtin — APPROVED** [review]
LGTM

File: Makefile [hunk 2]
```diff
@@ -2,10 +2,19 @@ install:
...
+	docker build -t eu.gcr.io/census-rm-ci/rm/census-rm-load-generator .
+
+delete-pod:
+	kubectl delete deploy load-generator
+
+apply-deployment:
+	kubectl apply -f load-generator-deployment.yml
+
+connect-to-pod:
+	 kubctl exec -it $(kubectl get pods --selector=app=load-generator -o jsonpath='{.items[*].metadata.name}') -- /bin/bash
```
Comment (gavinleeedwards) [inline Makefile]: Missing 'e' in `kubectl`

**Review by gavinleeedwards — CHANGES_REQUESTED** [review]

File: Makefile (hunk 2 above)
Comment (NickGrantX) [reply Makefile]: that's @AdamHawtin 's suggested change dammit

**Review by gavinleeedwards — APPROVED** [review]

**Pull request merged**